Configurations.md

```diff
@@ -2061,12 +2061,14 @@ use sit;
 
 ## `group_imports`
 
-Controls the strategy for how imports are grouped together.
+Controls the strategy for how consecutive imports are grouped together.
```

Comment:
Unfortunately I think `consecutive` may still be too ambiguous a description this context, especially for anyone just doing a passing read, as a reasonable person could assume (incorrectly) that it has connotations for whitespace as well. Would you mind extending this just a tad to cover that aspect more explicitly?

I'd offer a suggestion, but I can't come up with a different word that covers it any better, so maybe just something a bit more verbose?

E.g.

```suggestion
Controls the strategy for grouping sets of consecutive imports. Imports may contain newlines between imports and still be grouped together as a single set, but other statements between imports will result in different grouping sets.
```

Reply:
Agreed, thanks. Updated and squashed.